includes/editor-templates/navigator.php

```diff
@@ -29,14 +48,23 @@
 			?></span>
 		</button>
 	</div>
-	<div id="elementor-navigator__elements"></div>
+	<div id="elementor-navigator__elements" class="<?php echo esc_attr( $elements_list_class ); ?>"></div>
```

Comment:
`class="<?php echo esc_attr( $elements_list_class ); ?>"`

This can echo an empty class. Wrap this class in an `if` statement as well.